Storage/Storage/SQL/BrowserDB.swift

```diff
@@ -8,11 +8,22 @@ public enum QuerySort {
     case None, LastVisit, Frecency
 }
 
+public class DateRange {
+    let start: NSTimeInterval?
+    let end: NSTimeInterval?
+
+    public init(start: NSTimeInterval?, end: NSTimeInterval?) {
+        self.start = start
+        self.end = end
+    }
+}
+
 public enum FilterType {
     case ExactUrl
     case Url
     case Guid
     case Id
+    case DateRange
```

Comment:
This is valid and I support it, but I don't want to do this as part of this pull request.